rpcserver.go

```diff
@@ -878,6 +878,21 @@ func (s *rpcServer) WithdrawAccount(ctx context.Context,
 	}, nil
 }
 
+// hasPendingExtension returns if the accountKey is waiting for a batch to be
+// confirmed to extend the expiry height.
+func (s *rpcServer) hasPendingExtension(accountKey *btcec.PublicKey) bool {
+	if s.orderManager.HasPendingBatch() {
```

Comment:
Unfortunately this won't really work. The `orderManager.HasPendingBatch()` only returns true while a current batch is **ongoing** and hasn't been fully signed yet. As soon as the final batch TX is published this will be empty again.

I'm trying to remember, what was the original reason we didn't allow an account extension to be issued by the user if the account was still in the state `StatePendingBatch`? Because it could potentially throw the account out of sync with the server if another update TX was added?
Maybe I gave you some incorrect info there...

Reply:
:disappointed: thanks for catching this one